config.ts

```diff
@@ -1,8 +1,8 @@
 import { generateDeploymentConfig } from "scripts";
 const config = generateDeploymentConfig("conflict_forecast");
 
-config.google_drive.sheets_folder_ids =  ["[Add folder ID here]"];
-config.google_drive.assets_folder_ids = ["[Add folder ID here]"];
+config.google_drive.sheets_folder_ids = ["[17rK56zJj_W-1ceDlSPd2QFSpnwvApHh0]"];
+config.google_drive.assets_folder_ids = ["[1SzZiWN4hvv0snDdpd-BZYPjTC6RzC1uU]"];
```

Comment:
The inner set of square brackets should be removed, they're only present as part of the placeholder text. So the lines should look like this:

```ts
config.google_drive.sheets_folder_ids = ["17rK56zJj_W-1ceDlSPd2QFSpnwvApHh0"];
config.google_drive.assets_folder_ids = ["1SzZiWN4hvv0snDdpd-BZYPjTC6RzC1uU"];
```